Empire - The Last Battle/Assets/Unity/Scripts/ScriptableObjects/PlayerCards.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+[Serializable]
+public class PlayerCards : ScriptableObject
+{
+    public List<int> cardId;
+
+    // Use this for initialization
```

Comment:
Can you remove all of this stuff please. Scriptableobjects don't have start/update methods